module/helpers/chips.mjs

```diff
@@ -20,7 +20,11 @@ export class Chips {
   };
 
   static getColour(type) {
-    return this.colour(type);
+    const key = Object.keys(Chips.type)[
+      Object.values(Chips.type).indexOf(type)
+    ];
```

Comment:
Key order is not stable in JS objects, so `indexOf` might not necessarily return the one you're looking for. You need to instead run a search over the entire object and return `true` when the key's value is the right one:
```suggestion
    const key = Object.keys(Chips.type).find(searchKey => Chips.type[searchKey] === type);
```

`Array.find()` will return `undefined` if none of the keys match, but that was also true of your code.